monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -199,15 +199,10 @@ groups:
             Node {{ $labels.instance }} experiences packet errors > 1
             packet/s on interface {{ $labels.device }}.
 
-      # predict fs fill-up times
-      - alert: storage filling
+      - alert: storage filling up
```

Comment:
What about using a `for: 5m` to avoid false positives?

Reply:
I'm not sure if this is necessary here. The prediction is based on two days of data, there shouldn't be spikes that last just a few seconds.